schemas/Subscription.json

```diff
@@ -22,7 +22,10 @@
     },
     "target": {
       "description": "Target callback URI",
-      "$ref": "Iri.json"
+      "oneOf": [
+        { "$ref": "Iri.json" },
+        { "type": "string", "pattern": "^([*]{1})" }
```

Comment:
Ah, right, it should be `subject`, not `target`. And do these schemas support literal strings? Because that would be even cleaner.